RLTSquare/TrainingTask/etc/module.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<!--
+/**
+* @author RLTSquare
+* @copyright Copyright (c) 2022 RLTSquare (https://www.rltsquare.com/)
+* @package RLTSquare_TrainingTask
+*/
+ -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+    <module name="RLTSquare_TrainingTask"/>
```

Comment:
Undeclared module dependency - code uses Magento\Store\Model\ScopeInterface and Magento\Store\Model\StoreManagerInterface but module.xml has no dependency  declared on store module.

```
        <sequence>
            <module name="Magento_Store"/>
        </sequence>
```